packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -32,5 +33,17 @@ export const useButtonState = (draftState: ButtonState) => {
     }
   }
 
-  draftState.disabled = draftState['aria-disabled'] = draftState.disabled || draftState.loading;
+  draftState['aria-disabled'] = draftState.disabled;
+  draftState.disabled = draftState['aria-disabled'] && !draftState.disabledFocusable;
+
+  // Disallow keyboard events when component is disabled and eat events when disabledFocusable is set to true.
+  draftState.onKeyDown = (ev: React.KeyboardEvent<HTMLElement>) => {
+    const keyCode = getCode(ev);
+    if (draftState['aria-disabled'] && (keyCode === EnterKey || keyCode === SpacebarKey)) {
```

Comment:
Few lines above we are using `(ev.which === 20 || ev.which === 13)` 😕

Reply:
Good catch! I'll change the lines above!